clients/admin-ui/tailwind.config.js

```diff
@@ -2,7 +2,23 @@
 module.exports = {
   content: ["./src/**/*.{js,ts,jsx,tsx,mdx}"],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        // TODO: unify with existing Tailwind "gray" color
```

Comment:
Is there a ticket for this, by chance?

Reply:
I'm thinking this should be able to be done easily as part of the larger colors migration, but I can ticket it separately if you think that's a good idea.

Reply:
Is that what @Kelsey-Ethyca is working on? Just want to make sure it doesn't get lost in the shuffle.